node_modules/oae-authentication/lib/api.js

```diff
@@ -87,7 +87,6 @@ var _configUpdate = function(tenantAlias) {
     }
 };
 
-ConfigAPI.on('cleared', _configUpdate);
```

Comment:
I see what you mean, yea it could have an effect. The `update` event seems to have been handling that case, so this should be fine.